lib/db/feature-toggle-store.js

```diff
@@ -43,6 +52,34 @@ class FeatureToggleStore {
         return rows.map(this.rowToFeature);
     }
 
+    async getFeaturesClient() {
```

Comment:
Introducing a new method was a fast and easy fix. We should consider having the `getFeatures` method accept an arrays of fields to include in the response.

Reply:
Yeah, sounds like a good idea